### PR TITLE
ADR: Home Assistant Container as supported installation method

### PR DESCRIPTION
**Pull request by balloob**
Add an ADR for adding Home Assistant Container as a supported installation method as defined in #392.

### CODE REVIEW

File: adr/0013-home-assistant-container.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+There is a wide variety of containerization software available. From that perspective, Home Assistant will only actively document the use of Docker.
+
+### Required Expertise
+
+- **Installation**
+  This requires the user to have an existing system that can run Docker containers. Installation is either done by running a command from the Docker-cli or via a user interface (Synology, Portainer)
+
+* **Start when the system is started:** up to the container manager that the user uses.
+* **Run with full network access:** Default installation instructions prescribe net=host but the user can diverge. Usually happens when the user decides it’s not the Docker-way.
```
Comment (MartinHjelmare) [inline adr/0013-home-assistant-container.md:44]: We need to say what "Usually happens".

File: adr/0013-home-assistant-container.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+### Additional notes
+
+There is a wide variety of containerization software available. From that perspective, Home Assistant will only actively document the use of Docker.
+
+### Required Expertise
+
+- **Installation**
+  This requires the user to have an existing system that can run Docker containers. Installation is either done by running a command from the Docker-cli or via a user interface (Synology, Portainer)
+
+* **Start when the system is started:** up to the container manager that the user uses.
```
Comment (MartinHjelmare) [inline adr/0013-home-assistant-container.md:43]: What is up to the container manager? Do we mean that the home assistant container should start when the host system is started?

File: adr/0013-home-assistant-container.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* **Run with full network access:** Default installation instructions prescribe net=host but the user can diverge. Usually happens when the user decides it’s not the Docker-way.
+* **Access USB devices:** Responsibility of user.
```
Comment (MartinHjelmare) [inline adr/0013-home-assistant-container.md:45]: We need to explain this more. What about the access to USB devices is the responsibility of the user?

Comment (DubhAd) [inline adr/0013-home-assistant-container.md:45]: > Responsibility of user
to
> It is up to the user to ensure that all devices are correctly passed through to the container.

File: adr/0013-home-assistant-container.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+* **Maintenance Home Assistant installation**
+  If using the Docker-cli user needs to manually update the run command. If using a UI the user might be notified of an upgrade and automatically update. There is no rollback in case the instance does not come online after an update.
```
Comment (DubhAd) [inline adr/0013-home-assistant-container.md:48]: > If using the Docker-cli user needs
to
If using the Docker-cli the user needs

> notified of an upgrade and automatically update
Don't we normally advise against auto-updates?
> notified of an upgrade or automatically update - automatically applying updates may result in the system not coming back online.

File: adr/0013-home-assistant-container.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+### Additional notes
+
+There is a wide variety of containerization software available. From that perspective, Home Assistant will only actively document the use of Docker.
+
+### Required Expertise
+
+- **Installation**
+  This requires the user to have an existing system that can run Docker containers. Installation is either done by running a command from the Docker-cli or via a user interface (Synology, Portainer)
+
+* **Start when the system is started:** User is responsible for configuring this.
```
Comment (MartinHjelmare) [inline adr/0013-home-assistant-container.md:43]: I think it's still ambiguous what we're referring to in this bullet.

Maybe:

> The user is responsible for configuring the system to start the container when the system is started.

File: adr/0013-home-assistant-container.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+* **Access USB devices:** It is up to the user to ensure that all devices are correctly passed through to the container.
+
+* **Maintenance Home Assistant installation**
+  If using the Docker-cli the user needs to manually update the run command. If using a UI the user might be notified of an upgrade or automatically update – automatically applying updates may result in the system not coming back online. There is no rollback in case the instance does not come online after an update.
+
+  - **Python upgrades:** Included in the Home Assistant container
+  - **Installing Python dependencies:** Included in the Home Assistant container
+  - **Updating Home Assistant:** Included in the Home Assistant container
+
+- **Maintenance Operating System**
```
Comment (MartinHjelmare) [inline adr/0013-home-assistant-container.md:54]: > Maintenance of the Operating System

or

> Maintaining the Operating System

Comment (MartinHjelmare) [reply adr/0013-home-assistant-container.md:54]:
```suggestion
- **Maintaining the Operating System**
```

File: adr/0013-home-assistant-container.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+### Required Expertise
+
+- **Installation**
+  This requires the user to have an existing system that can run Docker containers. Installation is either done by running a command from the Docker-cli or via a user interface (Synology, Portainer)
+
+* **Start when the system is started:** User is responsible for configuring this.
+* **Run with full network access:** Default installation instructions prescribe net=host to be configured.
+* **Access USB devices:** It is up to the user to ensure that all devices are correctly passed through to the container.
+
+* **Maintenance Home Assistant installation**
```
Comment (MartinHjelmare) [inline adr/0013-home-assistant-container.md:47]:
```suggestion
* **Maintaining the Home Assistant installation**
```

File: adr/0013-home-assistant-container.md (hunk 5 above)
Comment (MartinHjelmare) [reply adr/0013-home-assistant-container.md:43]:
```suggestion
* **Start when the system is started:**
  The user is responsible for configuring the system to start the container when the system is started.
```

File: adr/0013-home-assistant-container.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+This is for running just the Home Assistant Core application on native OCI compatible containerization system. It does not provide the full Supervisor experience, and thus does not provide the Supervisor panel and add-ons.
```
Comment (DubhAd) [inline adr/0013-home-assistant-container.md:15]: > It does not provide the full Supervisor experience

This implies that it provides a partial Supervisor experience. Suggest removing "full" here

File: adr/0013-home-assistant-container.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+This is a general installation method that is recommended as an alternative to the Home Assistant One installation method. Due to the shared image with the Home Assistant One installation method, almost all documentation applies to the Home Assistant Container as well.
```
Comment (DubhAd) [inline adr/0013-home-assistant-container.md:17]: > alternative to the Home Assistant One installation method
> Due to the shared image with the Home Assistant One installation method

Replace One with TBC or get that ADR published first ;)

File: adr/0013-home-assistant-container.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+- Any Open Container Initiative (OCI) compatible containerization system.
+
+### Supported boards/hardware/machines
+
+- Machines of the following architectures: amd64, i386, armhf, aarch64, armv7
+
+### Supported Operating Systems and versions
+
+- Running Home Assistant Container, is only supported on Linux.
```
Comment (DubhAd) [inline adr/0013-home-assistant-container.md:31]: > Running Home Assistant Container, is only supported on Linux

Remove the comma

File: adr/0013-home-assistant-container.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+- Windows, and BSD installations (e.g., macOS and FreeBSD) are not supported.
```
Comment (DubhAd) [inline adr/0013-home-assistant-container.md:32]: > Windows, and BSD

Remove the comma

File: adr/0013-home-assistant-container.md (hunk 9 above)
Comment (balloob) [reply adr/0013-home-assistant-container.md:17]: Published: #395.

Marked hypervisors as tbd as that was blocking it.

**Pull request merged**